config/popular-email-domains.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I thought about putting it in convict, but opted not to because the domain list I had was over 120. The new list based on our users is about 20 domains log. I think this still might be to big for convict?